### DISCUSSION TITLE
Show SB Temp

### DISCUSSION THREAD

**Discussion by XestoX** [aldostools/webMAN-MOD#474] [Ideas]
Hey,

I read through some UART scripts and syscon command lists and found a interesting command that provide the Southbridge Temperature. Im a big Fan, airflow and temperature Freak / nerd I love messing around with these things and optimizing this factors on hard and software side. better paste, holes in case, delit SB on cok-001/002 and so on. Always trying to squizee out every half degree possible ^^

So would be awesome for me to see a befor delit and after delit of SB in temps. 

**Would it be possible to implement showing SB temp in any kind, it does not has to be in the START+SELECT popup, cause it would make this unnecessarily larger. but some menu option to show smb temp one time or fix it like START+SELECT?** 

If some one or you aldotools could ad this this would be some little dream to me that came true 👍 *.*

**Comment by aldostools** [comment]
Thank you for your feedback. Unfortunately I don't know how to read that parameter.

If someone have a snippet to get it, I could add that information easily to /cpursx.ps3 or to a web command that could be called through a custom combo

**Reply by XestoX** [reply]
I only know it out of this document about using UART to the SYSCON https://github.com/db260179/ps3syscon/blob/master/PS3-Uart-Guide.pdf (you find it on Page 18 of 20)

there it is a command called `tsensor 3`. Im a total noob at this topic that all I can provide sadly, I would do more if i could.

Maybe sharing this discussion to psx place or twitter to find someone in the community to be able to do that?

**Comment by aldostools** [comment]
There is a thread in PSX-PLACE about SYSCON. zecoxao and other guys report their progress in that thread.

You could ask there. Twiiter is not the best place to ask that kind of things ;-)

**Reply by XestoX** [reply]
thx for your reply, I will try this out and post this discussion here there. Hopeing the best ^^

[https://www.psx-place.com/threads/idea-to-add-a-feature-to-webman-mod-sb-temperature.32747/](url)

**Comment by XestoX** [comment]
"sandungas" from PSX-Place answered the following:

> I think is just needed to use the correct id of the thermal sensor, i was brainstorming about it in this post and other forum posts
> https://www.psx-place.com/threads/s...o-what-does-it-mean.26148/page-12#post-227211
> 
> The syscalls should be the same, is just we are only using the id's of the first 2 sensors (but sensor 3 doesnt exists, and southbridge seems to be id 4)

> The problem is we dont have access to the syscon commands (the commands that can be seen from the UART terminal) from gameOS
As far i understood is posible to install a syscon patch (using the oficial patch format) that contains an exploit that unlocks access to syscon commands... but as far i know the info required to implement this exploit has not been released yet
So we cant use the command tsensor 3 mentioned by XestoX in the github link you posted

> In the meantime the only thing we have are the syscalls, the same ones used to get the actual cell and rsx temperatures and fan speed, but using the sensor ID=3. It seems the sensors are given an ID starting by 0... so CELL is ID=0... RSX is ID=1... the ID=2 doesnt exists... and ID=3 seems to be the southbridge (is the fourth sensor)

So it seems to be the same code like for CELL/RSX Temps but just using the Temp seonsor No 4 instead of 1 and 2 (3 is not presend, I think it is one only unutilised in refference tools or so) 

[https://www.psx-place.com/threads/syscon-firmware-key-is-now-public-release-by-zecoxao-what-does-it-mean.26148/page-19#post-278286](url)

**Reply by aldostools** [reply]
Thank you for your feedback.

I tested changing the ID from 2 to 255 and using the same syscall used for CPU/RSX, but I didn't get values that looked like a real temperature.

Maybe if someone could show a PoC, then I could use the same code in webMAN.

**Reply by XestoX** [reply]
man so nice that you try that out, so thankfull for having you in the scene. I hope we will solve that but for now I already have to thank you to invest your time into this :)
